ikarus/controlroutines/pathfollowing.inl

```diff
@@ -26,47 +25,44 @@ requires(Impl::checkPathFollowingTemplates<NLS, PF, ASS>())
 ControlInformation PathFollowing<NLS, PF, ASS>::run() {
   ControlInformation info;
   auto& nonOp = nonLinearSolver_->nonLinearOperator();
-  this->notify(ControlMessages::CONTROL_STARTED, pathFollowingType_.name());
+  this->notifyListeners(ControlMessages::CONTROL_STARTED, pathFollowingType_.name());
```

Comment:
Should we keep `notify` or `notifyListeners`? I would like it shorted because otherwise it pollutes the classes even more but I also like the more destriptive nature of  `notifyListeners`.